application/system_processor/Cdl.py

```
@@ -0,0 +1,90 @@
+        self.goToPosition(cs[-2])
+
+    def stepOverForward(self, position):
+        '''
+            Step over function call forwards
+        '''
+        originalStackSize = len(self.decoder.callStacks[position])
+
+        while position < len(self.decoder.execution):
+            position = self.decoder.getNextExecutionPosition(position)
+
+            if position == None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)